playwright/src/test/java/com/microsoft/playwright/TestAutoClose.java

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.playwright;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * This test simply makes sure that all 4 main interfaces implement AutoCloseable. If it compiles, then it passes.
+ */
+public class TestAutoClose {
+  @Test
+  void shouldAllowUsingTryWithResources() throws Exception {
+    try (Playwright playwright = Playwright.create();
+         Browser browser = playwright.chromium().launch();
+         BrowserContext context = browser.newContext();
+         Page page = context.newPage()) {
+      page.navigate("about:config");
```

Comment:
I don't thing there is about:config in webkit, you could replace it with something more neutral like `assertEquals(2021, page.evaluate("() => 2021"));`